djstripe/checks.py

```diff
@@ -4,6 +4,8 @@
 from django.core import checks
 from django.utils.dateparse import date_re
 
+# todo add tests for this module
```

Comment:
Can you avoid leaving TODOs in the code like this. ONLY leave a TODO if it's immediately relevant to the code below; but this kind specifically will just get lost to time and forgotten.